gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -628,6 +628,7 @@ public void testInitializeThrowsWhenCredentialsNotFound() throws URISyntaxExcept
 
     assertThat(thrown).hasMessageThat().startsWith("No enum constant ");
     assertThat(thrown).hasMessageThat().contains("AuthenticationType.INVALID_AUTH_TYPE");
+    ghfs.close();
```

Comment:
This wont close if there is test failure, correct?

Reply:
same as below.